lightning/src/ln/channelmanager.rs

```diff
@@ -4581,6 +4589,29 @@ impl<M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelManager<M, T, K, F
 		self.our_network_pubkey.clone()
 	}
 
+	fn handle_monitor_update_completion_actions<I: IntoIterator<Item=MonitorUpdateCompletionAction>>(&self, actions: I) {
+		let mut actions = actions.into_iter().peekable();
+		if actions.peek().is_some() {
```

Comment:
suggestion: early terminate rather than adding indentation, ie: `if actions.peek().is_none() { return; }`
```
diff --git a/lightning/src/ln/channelmanager.rs b/lightning/src/ln/channelmanager.rs
index 5269e910..da489ed0 100644
--- a/lightning/src/ln/channelmanager.rs
+++ b/lightning/src/ln/channelmanager.rs
@@ -4562,7 +4562,7 @@ impl<M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelManager<M, T, K, F

        fn handle_monitor_update_completion_actions<I: IntoIterator<Item=MonitorUpdateCompletionAction>>(&self, actions: I) {
                let mut actions = actions.into_iter().peekable();
-               if actions.peek().is_some() {
+               if actions.peek().is_none() { return; }
                        let mut pending_events = self.pending_events.lock().unwrap();
                        for action in actions {
                                match action {
@@ -4580,7 +4580,6 @@ impl<M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelManager<M, T, K, F
                                        },
                                }
                        }
-               }
        }

        /// Handles a channel reentering a functional state, either due to reconnect or a monitor